app-modules/meeting-center/src/Filament/Resources/CalendarEventResource/Pages/ListCalendarEvents.php

```diff
@@ -156,7 +159,13 @@ public function table(Table $table): Table
                     DeleteBulkAction::make(),
                 ]),
             ])
-            ->modifyQueryUsing(fn (Builder $query) => $query->orderBy('starts_at'));
+            ->modifyQueryUsing(function (Builder $query): Builder {
+                /** @var User $user */
+                $user = auth()->user();
+
+                return $query->whereRelation('calendar', 'user_id', $user->id);
+            })
+            ->defaultSort('starts_at', 'desc');
```

Comment:
Would this have previously been scoped to the authenticated user's calendar events?

Reply:
So the change to `modifyQueryUsing` now makes it so you only see your events, which I believe is what Joe expects.

I moved the sorting stuff to `defaultSort` because, though what you had works, it would ALWAYS make that sorting happing, so clicking the other sorting buttons did nothing to the list.

This makes it so starts_at desc is the default, but a user COULD change it if they wanted.